src/mobile/navigation/component.js

```diff
@@ -177,11 +177,12 @@ Controller.prototype.init = function (element) {
     if (!datatarget) {
       throw new Error('Missing datatarget');
     }
-    const slideIn = $(datatarget);
-    if (slideIn.length != 1) {
+    const slideInFound = $.find(datatarget);
+    if (slideInFound.length != 1) {
       throw new Error('Wrong slideIn');
     }
 
+    const slideIn = $(slideInFound);
```

Comment:
Ok, that's coherent to keep using jquery as we are using jquery in the rest of the file.
But we could also not using jquery anymore. That comes from the last decade :rofl:

Reply:
Effectively, I do the minimal change :-)